src/main/resources/CU-ApplicationResources.properties

```diff
@@ -248,7 +247,7 @@ error.po.not.fully.received.and.invoiced=PO indicates receiving is required and
 
 	errors.add.new.note.sendToVendor.noAtt=Send to Vendor note must have an Attachment.
 	errors.exceed.sq.number.of.att.limit=Failed to {0} because the number of 'Send to Vendor' Attachments exceeds Sciquest limit of {1}. 
-	errors.att.filesize.over.limit=Attachment "{0}" file size over limit of {1} MB.
+	errors.att.send.to.vendor.filesize.over.limit=Attachment "{0}" file size over limit of {1}.
```

Comment:
Just to confirm, is this property being read properly by KFS despite its leading whitespace? If it's still working as expected, then I'm okay with cleaning up that whitespace in a future ticket, since I'm planning to create one anyway for dealing with various duplicate properties in this file.

Reply:
I verified locally that the error message is displaying as desired before and after the most recent commit that was made.

![image](https://user-images.githubusercontent.com/33495813/235949543-62217234-0ffa-4ac2-b371-3137fb04197a.png)

Reply:
Thanks for confirming. Your changes can proceed as-is and I'll create a follow-up user story to clean up the properties file further.